docs/sdk/wrapped-keys/storing-wrapped-key-metadata.md

```diff
@@ -0,0 +1,333 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Storing Wrapped Key Metadata
+
+This guide covers the `storeEncryptedKeyMetadata` function from the Wrapped Keys SDK. For an overview of what a Wrapped Key is and what can be done with it, please go [here](./overview.md).
+
+The `storeEncryptedKeyMetadata` function allows you to manually initialize a Wrapped Key by providing the required Wrapped Key metadata. Lit will store the provided metadata in a private instance of DynamoDB, effectively registering the Wrapped Key for use.
+
+This method is useful for when you would like to implement your own method of generating a private key and encrypting it with Lit network's public BLS key. This could be code completely ran on your own infrastructure, or this could be a custom Lit Action that you've implemented. The stored metadata can then later be used with the other Wrapped Keys SDK methods (e.g. [signMessageWithEncryptedKey](./sign-message.md) or [signTransactionWithEncryptedKey](./sign-transaction.md)), or custom Lit Actions you create to sign data with a Wrapped Key.
```

Comment:
```suggestion
This method is useful for when you would like to implement your own method of generating a private key and encrypting it with Lit network's public BLS key. This could be code completely ran on your own infrastructure, or this could be a custom Lit Action that you've implemented. The stored metadata can then later be used with custom Lit Actions you create to sign data with a Wrapped Key.
```

Reply:
If you store the metadata can you not use it with the SDK methods for signing?

Reply:
The idea is that if you wanna use Custom Wrapped Keys then don't use these predefined functions and directly use `executeJs`